walkthroughs/1-ionic-showcase/walkthrough.adoc

```diff
@@ -0,0 +1,339 @@
+* From the terminal, ensure you have the correct namespace selected.
+
+....
+oc project <project where template was provisioned>
+....
+
+* Update the ionic-showcase-server deployment to add the `+MQTT_HOST+`
+variable.
+
+....
+oc set env dc/ionic-showcase-server MQTT_HOST="$(oc get addressspace showcase -o jsonpath='{.status.endpointStatuses[?(@.name=="messaging")].serviceHost}')"
+....
+
+At this point, the showcae server is provisioned and the logs from the
+ionic-showcase-server pod will include the output
+`+connected to messaging service+`.
+
+. Try out the link:{route-ionic-showcase-server-host}/graphql[GraphQL playground, window="_blank"].
+. Paste in the following query/mutation into the text edit:
++
+.Sample GraphQL Queries
+----
+query listTasks {
+  allTasks {
+    title,
+    description,
+    id
+  }
+}
+
+mutation createTask {
+  createTask(title: "complete the walkthrough", description: "complete the mobile walk through") {
+    title,
+    description,
+    version,
+    id
+  }
+}
+----
++
+and try execute them. Feel free to explore the schema and add more queries and mutations.
+
+[time=5]
+== Create Mobile Application in Mobile Developer Console
+
+The Mobile Developer Console (MDC) is the starting point for developing and managing mobile apps on Open Shift.
+Console will allow mobile developers to view avaiable mobile services, connect them them to their mobile apps and get the mobile app configurations.
+The MDC helps you represent and manage your apps in Open Shift, it shows you the services that are available to you, and it helps you integrate your apps with those services using a concept called service bindings. The end result is a configuration file that can be dropped into your project. 
+The client SDKs will use this configuration file to to talk to the all Mobile Services on OpenShift
+
+Developer Console is already available in cluster and accessible under following url: 
+
+link:{route-mdc-server-host}[Mobile Developer Console, window="_blank"]
+
+=== Steps
+
+. Go to link:{route-mdc-server-host}[Mobile Developer Console, window="_blank"], and you can login with your OpenShift credential.
+MDC panel will allow us to create new Mobile Applications and connect (bind) them with all available Mobile Developer services
+. Click on the `Create Mobile App` button to create a new app. 
+. Put any name for the mobile application. For example `demo`.
+. When the app is created, click on it and it will bring you to the configure page. On the left hand side, 
+  you will see Wathe instructions to add the SDKs to your client app. On the right hand side you will see the content of the configuration file to add to a client app. There should be no services listed in the configuration file.
+. Go to the `Mobile Services` tab, and you should see there are a few services listed in the `Unbound Services` section.
+. From `Unbound Services` select `DataSync
+. Skip binding
```

Comment:
@wtrocki  not sure I understand what you're trying to do here

Reply:
This is related to todo here. I'm not sure if we should bind all servives here or bind them again in optonal steps